doc/settings.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$id": "https://aka.ms/winget-settings.schema.json",
+  "$schema": "https://json-schema.org/draft/2019-09/schema#",
+  "title": "Microsoft's Windows Terminal Settings Profile Schema",
+  "definitions": {
+    "Source": {
+      "description": "Source repository settings",
+      "type": "object",
+      "properties": {
+        "autoUpdateIntervalInMinutes": {
+          "description": "Number of minutes before source update",
+          "type": "integer",
+          "default": 5,
+          "minimum": 0,
+          "maximum": 65535
```

Comment:
65535 seems like an arbitrary number, maybe a value equivalent to 30 days(like 43200) or something meaningful. Our code does not enforce a specific max now though